docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -0,0 +1,205 @@
+- the square footage of the house
+- the number of bedrooms and bathrooms
+- the zip code
+
+The label is the historical house price for that row of square footage, bedroom, and bathroom values and zip code.
+
+![Table showing rows and columns of house price data with features consisting of size rooms zip code and price label](media/model-builder-data.png)
+
+### Data formats
+
+Model Builder places the following limitations on the data:
+
+- Data must be stored in a file (.csv or .tsv with a header row), or in a SQL server database.
+- A limit of 1 GB on the training dataset
+- SQL server has a limit of 100,000 rows for training
+- Data from SQL server is copied from the server to your local machine before training
+
+### Example datasets
+
+If you don't have your own data yet, try out one of these datasets:
+
+|Scenario|ML Task|Data|Label|Features|
+|-|-|-|-|-|
+|Price prediction|regression|[taxi fare data](https://github.com/dotnet/machinelearning-samples/blob/master/datasets/taxi-fare-train.csv)|Fare|Trip time, distance|
+|Anomaly detection|binary classification|[product sales data](https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/getting-started/AnomalyDetection_Sales/SpikeDetection/Data/product-sales.csv)|Product Sales|Month|
+|Sentiment analysis|binary classification|[website comment data](https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/SentimentAnalysis/Data/wikiDetoxAnnotated40kRows.tsv)|Label (0 when negative sentiment, 1 when positive)|Comment, Year|
+|Fraud detection|binary classification|[credit card data](https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/getting-started/BinaryClassification_CreditCardFraudDetection/CreditCardFraudDetection.Trainer/assets/input/creditcardfraud-dataset.zip)|Class (1 when fraudulent, 0 otherwise)|Amount, V1-V28 (anonymized features)|
+|Text classification|multiclass classification|[GitHub issue data](https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/end-to-end-apps/MulticlassClassification-GitHubLabeler/GitHubLabeler/Data/corefx-issues-train.tsv)|Area|Title, Description|
+
+## Train
+
+Once you select your scenario, data, and label, Model Builder trains the model.
+
+### What is training?
+
+Training is an automatic process by which Model Builder teaches your model how to answer questions for your scenario. Once trained, your model can make predictions with input data that it has not seen before. For example, if you are predicting house prices and a new house comes on the market, you can predict its sale price.
+
+Model Builder uses automated machine learning (AutoML), which automatically explores different machine learning algorithms and settings to find the one that best suits your scenario. Model Builder does not require any input or tuning from you. No prior expertise for ML is required.
```

Comment:
This seems redundant as it was stated previously and could be removed for brevity.